packages/common/src/components/utils/ExpandableTextWithTitle.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect, useState } from 'react'
+import styled from 'styled-components'
+import { ChevronDownIcon, ChevronUpIcon } from '../../images'
+
+const StyledLink = styled.a`
+  display: flex;
+  white-space: nowrap;
+`
+
+const TitleBox = styled.div`
+  display: flex;
+  justify-content: space-between;
+`
+
+const Root = styled.div`
+  width: 100%;
+`
+
+interface Props {
+  text: string
+  title: string
+}
+
+export const ExpandableTextWithTitle: React.FC<Props> = ({ text, title }) => {
+  const [expanded, setExpanded] = useState(false)
+
+  useEffect(() => {
+    setExpanded(false)
+  }, [text])
+
+  return (
+    <Root>
+      <TitleBox>
+        <p>{title}</p>
+        <div>
+          <StyledLink href="#" onClick={() => setExpanded(!expanded)}>
```

Comment:
Should be a button element instead of anchor. It's currently disabled in common for legacy reasons, but we might want to enable this rule: [anchor-is-valid](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/93f78856655696a55309440593e0948c6fb96134/docs/rules/anchor-is-valid.md)